src/pages/EditSplitBillPage.js

```diff
@@ -130,6 +138,30 @@ function EditSplitBillPage({route, transaction, draftTransaction}) {
         );
     }
 
+    if (fieldToEdit === CONST.EDIT_REQUEST_FIELD.CATEGORY) {
+        return (
+            <EditRequestCategoryPage
+                defaultCategory={transactionCategory}
+                policyID={lodashGet(report, 'policyID', '')}
+                onSubmit={(transactionChanges) => {
+                    setDraftSplitTransaction({category: transactionChanges.category.trim()});
```

Comment:
Coming from https://github.com/Expensify/App/issues/37146, this has caused a bug where you can't un-select a category when editing split bill